indico/web/client/js/react/components/PersonDetailsModal.jsx

```diff
@@ -141,15 +147,20 @@ export default function PersonDetailsModal({
           <Translate as="label">First Name</Translate>
           <FinalInput name="firstName" />
         </Form.Field>
-        <Form.Field>
+        <Form.Field required>
```

Comment:
This isn't required on purpose. In some cases you may not have a full first and last name for a person.